department-of-veterans-affairs/caseflow
# Adding non-availability data structures

Connects #5709 

### Description
This PR adds a table for non-availability days.

### Testing Plan
1. Run the migration
1. In the rails console run: 
    1. `RoNonAvailability.new(schedule_period_id: SchedulePeriod.first.id, date: Date.today, object_identifier: 'RO183').save!`
    1. `CoNonAvailability.new(schedule_period_id: SchedulePeriod.first.id, date: Date.today, object_identifier: 'CO').save!`
    1. `JudgeNonAvailability.new(schedule_period_id: SchedulePeriod.first.id, date: Date.today, object_identifier: 'BVAGWASHINGTON').save!`
    1. `NonAvailability.all`
1. Confirm those commands run correctly, the non-availability days are properly saved, and the non-availability days can be fetched.
